// Chapter 9: Error Handling

mod panicking;
mod recoverable_with_result;
mod when_to_panic;

fn main() {
    // panicking::run();
    // recoverable_with_result::run();
    when_to_panic::run();
}
